src/library.js

```diff
@@ -1328,6 +1351,52 @@ const createPilingJs = (rootElement, initOptions = {}) => {
       stateUpdates.add('layout');
     }
 
+    if (state.itemOpacity !== newState.itemOpacity) {
+      if (newState.itemOpacity) {
```

Comment:
What happens in the case `newState.itemOpacity` is `false` (i.e., `null`, `undefined`, etc.)? I think in that case we need to set `item.alpha` back to its default state, i.e., `1.0`.